BraintreeCore/src/main/java/com/braintreepayments/api/TokenizationClient.java

```diff
@@ -25,56 +22,14 @@
         this.braintreeClientRef = braintreeClientRef;
     }
 
-    /**
-     * Create a {@link PaymentMethodNonce} in the Braintree Gateway.
-     * <p>
-     * On completion, returns the {@link PaymentMethodNonce} to {@link TokenizeCallback}.
-     * <p>
-     * If creation fails validation, {@link TokenizeCallback#onResult(org.json.JSONObject, Exception)}
-     * will be called with the resulting {@link ErrorWithResponse}.
-     * <p>
-     * If an error not due to validation (server error, network issue, etc.) occurs, {@link
-     * TokenizeCallback#onResult(org.json.JSONObject, Exception)} will be called with the {@link Exception} that occurred.
-     *
-     * @param paymentMethod {@link PaymentMethod} for the {@link PaymentMethodNonce}
-     *        to be created.
-     * @param callback {@link TokenizeCallback}
-     */
-    void tokenize(final PaymentMethod paymentMethod, final TokenizeCallback callback) {
+    void tokenizeGraphQL(final JSONObject tokenizePayload, final TokenizeCallback callback) {
         final BraintreeClient braintreeClient = braintreeClientRef.get();
         if (braintreeClient == null) {
             return;
         }
 
-        paymentMethod.setSessionId(braintreeClient.getSessionId());
-        braintreeClient.getConfiguration(new ConfigurationCallback() {
-            @Override
-            public void onResult(@Nullable Configuration configuration, @Nullable Exception error) {
-                if (configuration != null) {
-                    if (paymentMethod instanceof GraphQLTokenizable &&
-                            configuration.isGraphQLFeatureEnabled(Features.TOKENIZE_CREDIT_CARDS)) {
-                        tokenizeGraphQL(braintreeClient, (GraphQLTokenizable) paymentMethod, callback);
-                    } else {
-                        tokenizeRest(braintreeClient, paymentMethod, callback);
-                    }
```

Comment:
We decided to move this logic to `CardClient` since `Card` is technically the only feature using GraphQL at the moment. This also allowed us to make the `buildJSON` method `package-private`.